docs/faq.md

```diff
@@ -72,3 +72,9 @@ description: Get to know more about Next.js with the frequently asked questions.
 
   <p>As we were researching options for server-rendering React that didn’t involve a large number of steps, we came across <a href="https://github.com/facebookarchive/react-page">react-page</a> (now deprecated), a similar approach to Next.js by the creator of React Jordan Walke.</p>
 </details>
+
+<details>
+  <summary>Can I make a Next.js Progressive Web App?</summary>
+   <p>Yes! Here's an <a href="https://github.com/vercel/next.js/tree/canary/examples/progressive-web-app">example</a>.</p>
+
```

Comment:
```suggestion
```

Reply:
@lfades Are you sure you want to remove it? It was mentioned in issue to add this example to FAQ.

Reply:
@eKhattak My change is only removing a new line that shouldn't be there